pkgs/applications/graphics/displaycal/default.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, python3
+, xorg
+, ffmpeg
+, argyllcms
+, gsettings-desktop-schemas
+, gtk3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "displaycal";
+  version = "3.9.10";
```

```suggestion
  version = "3.9.10";
  format = "setuptools";
```
at some point this will become mandatory :)